sprout.yaml

```diff
@@ -178,13 +178,20 @@ resources:
             exec > >(tee -a /var/log/clearwater-heat-sprout.log) 2>&1
             set -x
 
-            # Set up the signaling network namespace
+            # Set up the signaling network namespace on each boot.
+            cat >/etc/rc.local <<EOF
```

Comment:
Can we be sure /etc/rc.local doesn't already exist?  Should we append to it rather than overwriting?